src/Texture2D.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio
+** File description:
+** Texture2D
+*/
+
+#ifndef TEXTURE2D_HPP_
+#define TEXTURE2D_HPP_
+
+#include <iostream>
+#include <memory>
+
+struct Texture;
+struct Rectangle;
+struct Vector2;
+struct Color;
+
+typedef Texture Texture2D;
+
+namespace indie {
+    class Texture {
+        public:
+            Texture(const std::string &fileName);
+            ~Texture();
+
+            /**
+             * @brief Load texture from file into GPU memory (VRAM)
+             * @param fileName The file name
+             */
+            void LoadTexture(const std::string &fileName);
+
+            /**
+             * @brief Unload texture from GPU memory (VRAM)
```

Comment:
You may use "///" instead of "/**"
```suggestion
             /// @brief Unload texture from GPU memory (VRAM)
```